integrated_channels/integrated_channel/management/commands/__init__.py

```diff
@@ -0,0 +1,101 @@
+"""
+Enterprise Integrated Channel management commands.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django.core.management.base import CommandError
+from django.utils.translation import ugettext as _
+
+from enterprise.models import EnterpriseCustomer
+from integrated_channels.sap_success_factors.models import SAPSuccessFactorsEnterpriseCustomerConfiguration
+
+
+# Import djcelery, or stub it if not available.
+try:
+    from djcelery.celery import task as celery_task
+except ImportError:
+    def celery_task(func):
+        """Use a no-op decorator if djcelery is not available."""
+        def no_delay(*args, **kwargs):
+            """
+            Provides the celery.task.delay function, which can be used to spawn an asynchronous celery task.
+            Here, it just calls the function.
+            """
+            func(*args, **kwargs)
+        func.delay = no_delay
+        return func
+
+# Mapping between the channel code and the channel configuration class
+INTEGRATED_CHANNEL_CHOICES = {
+    channel_class.channel_code(): channel_class
+    for channel_class in (SAPSuccessFactorsEnterpriseCustomerConfiguration, )
+}
+
+
+class IntegratedChannelCommandMixin(object):
+    """
+    Contains common functionality for the IntegratedChannel management commands.
+    """
+    def add_arguments(self, parser):
+        """
+        Adds the optional arguments: ``--enterprise_customer``, ``--channel``
+        """
+        parser.add_argument(
+            '--enterprise_customer',
+            dest='enterprise_customer',
+            default=None,
+            metavar=_('ENTERPRISE_CUSTOMER_UUID'),
+            help=_('Transmit data for only this EnterpriseCustomer. '
+                   'Omit this option to transmit to all EnterpriseCustomers with active integrated channels.'),
+        )
+        parser.add_argument(
+            '--channel',
+            dest='channel',
+            default='',
+            metavar=_('INTEGRATED_CHANNEL'),
+            help=_('Transmit data to this IntegrateChannel. '
+                   'Omit this option to transmit to all configured, active integrated channels.'),
+            choices=INTEGRATED_CHANNEL_CHOICES.keys(),
+        )
+
+    def get_integrated_channels(self, options):
```

Comment:
@pomegranited nit: this method is 40 lines lon - probably a good time to split it. How about extracting the for loop at the end to a dedicated method?